src/components/multisigParameters.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<a-card title="Multisignature parameters" style="margin-top: 14px">
+		Addresses:
+		<li
+			v-for="ADDRESSES in mparams.addresses"
+			:key="ADDRESSES.address"
+			class="list_addresses"
+		>
+			<CheckCircleTwoTone v-if="ADDRESSES.signed" twoToneColor="LightGreen" />
+			<CloseCircleTwoTone twoToneColor="red" v-else />
+			{{ ADDRESSES.address }}
+		</li>
+		<p style="margin-top: 10px">
+			Threshold: {{ mparams.threshold }}/{{ mparams.addresses.length }}
+		</p>
+		<p>Version: {{ mparams.version }}</p>
+	</a-card>
+</template>
+
+<script lang="ts">
+import { MultisignatureParam } from "@/types";
+import { CheckCircleTwoTone, CloseCircleTwoTone } from "@ant-design/icons-vue/";
+import algosdk from "algosdk";
+import { defineComponent } from "vue";
+
+export default defineComponent({
+	name: "Multiparameters-box",
+	props: ["inputBase64"],
+	components: {
+		CheckCircleTwoTone,
+		CloseCircleTwoTone,
+	},
+	data() {
+		const mparams: MultisignatureParam = {
+			addresses: [],
+			version: 0,
+			threshold: 0,
+		};
+
+		return {
+			mparams,
+			input: "base64",
+		};
+	},
+	methods: {
+		MultiParams() {
+			this.input = this.inputBase64;
```

Comment:
shouldn't this be updated by v:model?

Reply:
No, it should not. Because `inputBase64` is in type `any` but I want to specify the `input` to be string. It was props also so I did not user v:model [https://vuejs.org/guide/essentials/component-basics.html#passing-props]